spec/services/exports/lettings_log_export_service_spec.rb

```diff
@@ -217,7 +217,7 @@ def replace_record_number(export_template, record_number)
 
       it "creates a logs export record in a database with correct time" do
         expect { export_service.export_xml_lettings_logs }
-          .to change(LogsExport, :count).by(1)
+          .to change(LogsExport, :count).by(3)
```

Comment:
I assume this is 3 because we're also exporting the next years data? Do we want to?